fs-ansible/roles/lustre/tasks/format.yaml

```diff
@@ -40,8 +40,15 @@
         state: present
       loop: "{{ mdts.keys() }}"
 
+    - name: Wait for MDT partition to appear in /dev
+      wait_for:
+        path: "/dev/{{ item.key }}p1"
+        timeout: 30
+        sleep: 5
+      loop: "{{ mdts|dict2items }}"
```

Comment:
:+1:

Reply:
actually a slight consitency thing, normally been doing mtds.keys() and {{ item }}

Reply:
Thanks yes I did end up using mdts.keys(), see a more recent commit.